apis/nucleus/src/components/ActionsToolbarMore.jsx

```diff
@@ -20,7 +20,13 @@ const MoreItem = ({ item, onActionClick = () => {} }) => {
     onActionClick();
   };
   return !hidden ? (
-    <MenuItem title={item.label} onClick={handleClick} disabled={disabled}>
+    <MenuItem
+      title={item.label}
+      onClick={handleClick}
+      disabled={disabled}
+      aria-expanded="true"
```

Comment:
This should be a dynamic value. I think there is a state that determines whether or not the more dropdown is open or not, change to that